addPost.php

```diff
@@ -2,34 +2,32 @@
     require_once 'src/Models/BlogModel.php';
     require_once 'src/connectToDb.php';
 
-    session_start();
-    $db = connectToDb();
-
     if (!isset($_SESSION['userid'])) {
```

Comment:
this line will now error because the `session_start()` hasnt been called yet. move it back to where it was.